example/pandas_scrapper.py

```diff
@@ -0,0 +1,20 @@
+from obsei.source.pandas_source import (
```

Comment:
let's change the name of file. As this is not a scrapper.

Reply:
Sure sir I'll name this script pandas_source_example.py? That better ?

Reply:
Yes this would be better name